node/node.go

```diff
@@ -196,6 +196,18 @@ func (c *Node) ReceivedVouchers() <-chan payments.Voucher {
 	return c.receivedVouchers
 }
 
+// CreateVoucher creates a voucher for the given channelId and amount and returns it.
+// It is the responsibility of the caller to send the voucher to the payee.
```

Comment:
Subtlety around increment vs running total here. Maybe:

```suggestion
// CreateVoucher creates and returns a voucher for the given channelId which increments the redeemable balance by amount.
// It is the responsibility of the caller to send the voucher to the payee.
```